backend/src/main/java/com/htecgroup/skynest/controller/LambdaController.java

```diff
@@ -1,24 +1,120 @@
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "[\"UPLOAD_FILE_TO_EXTERNAL_SERVICE_LAMBDA\","
+                                + "\"SOME_OTHER_LAMBDA\"]")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized request",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Access denied\"],"
+                                + " \"status\": \"401\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+      })
   @GetMapping
   public List<LambdaType> getAllLambdas() {
     return Arrays.stream(LambdaType.values()).collect(Collectors.toList());
   }
+
+  @Operation(summary = "Activate lambda for bucket")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "Lambda successfully activated for given bucket",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = LambdaType.class),
+                  examples = {@ExampleObject(value = "true")})
+            }),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized request",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Access denied\"],"
+                                + " \"status\": \"401\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "409",
+            description = "Lambda already activated",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Lambda is already activated for that bucket.\"],"
+                                + " \"status\": \"409\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+      })
+  @PutMapping("/bucket/{bucketId}/activate")
+  public ResponseEntity<Boolean> activateLambdaForBucket(
```

Comment:
You don't need to return a boolean response, instead you can return 204 status code with no content. If I am not mistaking this was already implemented in some delete endpoint by @DraganMitrasinovicHtec